### ISSUE TITLE
Is it possible to shoot through Flag..

### ISSUE THREAD

**Issue by Punjara** [assaultcube/AC#275]
Lots of time, people get killed because they were shooting enemy through flag...lol which is not possible. So if it is possible to implement it to shoot through the flag, it would be nice.

**Comment by BrutalSystem** [comment]
> Lots of time, people get killed because they were shooting enemy through flag...lol which is not possible. So if it is possible to implement it to shoot through the flag, it would be nice.

It's a feature now, as far as I know. because those who use alternative pickups have an advantage over those who use the original models.

let's say I changed the health of my pickup, now my pickup is a tiny grain of sand, if someone tries to use the original models to get protection behind, there will be no protection, because my shots go through.

me who asked stef to change that.

another detail: probably AC does not check the rotation of the pickups, so this is another reason why this has been added.

so the game is fairer for everyone.
